Error: In order to serve as an `id`-reference for `aria-labelledby`, the `label` must have an `id` attribute, not `for`. The `for` attribute is itself an `id`-reference to an  associated form control, which is not applicable in this case.
Error: Checkboxes in a group should use a `name` value that ends with "[]" to allow server-side code to collect the values as an array (or similar).
diff --git a/output/Checkbox Group with Accessible Description/4/57199ad9-8ae4-4ea5-8138-31e5d4400d0f.html b/output/Checkbox Group with Accessible Description/4/57199ad9-8ae4-4ea5-8138-31e5d4400d0f.html
index 4dcf0ac..a410a6f 100644
--- a/output/Checkbox Group with Accessible Description/4/57199ad9-8ae4-4ea5-8138-31e5d4400d0f.html	
+++ b/output/Checkbox Group with Accessible Description/4/57199ad9-8ae4-4ea5-8138-31e5d4400d0f.html	
@@ -1,17 +1,17 @@
 <div>
-  <label for="fruits">What fruits do you like?</label>
+  <label id="fruits">What fruits do you like?</label>
   <div id="fruits-description">You can change this later</div>
   <div role="group" aria-labelledby="fruits" aria-describedby="fruits-description">
     <div>
-      <input type="checkbox" id="bananas" name="fruits" value="bananas">
+      <input type="checkbox" id="bananas" name="fruits[]" value="bananas">
       <label for="bananas">Bananas</label>
     </div>
     <div>
-      <input type="checkbox" id="oranges" name="fruits" value="oranges">
+      <input type="checkbox" id="oranges" name="fruits[]" value="oranges">
       <label for="oranges">Oranges</label>
     </div>
     <div>
-      <input type="checkbox" id="apples" name="fruits" value="apples">
+      <input type="checkbox" id="apples" name="fruits[]" value="apples">
       <label for="apples">Apples</label>
     </div>
   </div>
